config/300-topic.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+ name: topics.pubsub.cloud.run
+ labels:
+   events.cloud.run/release: devel
+   events.cloud.run/crd-install: "true"
+spec:
+  group: pubsub.cloud.run
```

Comment:
Rather than exposing topic and pull_subscriptions as public interface, we should expose the importer as the public resource (which will leverage pull_subscription). And keep pull_subscription and topic as internal objects.

Reply:
they are part of pubsub.cloud.run, you can choose to view that as internal.

Reply:
The `Channel` type is also part of pubsub.cloud.run. Will that move to a different api group?